src/components/display/fewlines/IdentityOverview/IdentityOverview.tsx

```diff
@@ -15,6 +15,90 @@ type IdentityOverviewProps = {
   identity: Identity;
 };
 
+const PrimaryConfirmationBox = (
```

Comment:
Can you move these two components into a dedicated file?

Reply:
OK done 65e8036